packages/dev/core/src/Collisions/gpuPicker.ts

```diff
@@ -92,6 +92,13 @@ export class GPUPicker {
         return this._pickingInProgress;
     }
 
+    /**
+     * Gets the default render material used by the picker.
+     */
+    public get defaultRenderMaterial(): Nullable<ShaderMaterial> {

```

Comment:
Does this have to be exposed as `ShaderMaterial` (rather than something more abstract, like `Material`)?

Reply:
I think so because the internal object is a shaderMaterial and this will also let the users control the gpupicker rendering more if they want to